crates/consistent-hashing/README.md

```diff
@@ -0,0 +1,60 @@
+# Consistent Hashing
+
+Consistent hashing maps keys to a changing set of nodes (shards, servers) so that when nodes join or leave, only a small fraction of keys move. It is used in distributed caches, databases, object stores, and load balancers to achieve scalability and high availability with minimal data reshuffling.
+
+Common algorithms
+- [Consistent hashing](https://en.wikipedia.org/wiki/Consistent_hashing) (hash ring with virtual nodes)
+- [Rendezvous hashing](https://en.wikipedia.org/wiki/Rendezvous_hashing)
+- [Jump consistent hash](https://en.wikipedia.org/wiki/Jump_consistent_hash)
```

Comment:
Wikipedia link didn't work for me, maybe this: https://arxiv.org/pdf/1406.2294 ?